codemods/jscodeshift/next/13/remove-get-static-props/README.md

```diff
@@ -1 +1,39 @@
 # remove-get-static-props
+In the `pages` directory, the `getStaticProps` function is used to pre-render a page at build time by fetching data passing it down to the entire page as it is being generated during the build. However, the `app` directory data fetching will default to `cache: 'force-cache'`, which will cache the request data until manually invalidated.
+
+The `remove-get-static-props` codemod replaces the `getStaticProps()` data fetching function in the `pages` directory with the new data fetching API in the `app` directory.
+
+For example:
+```jsx
+// `pages` directory
+ 
+export async function getStaticProps() {
+  const res = await fetch(`https://...`);
+  const projects = await res.json();
+ 
+  return { props: { projects } };
+}
+ 
+export default function Index({ projects }) {
+  return projects.map((project) => <div>{project.name}</div>);
+}
+```
+
+Transforms into:
```

Comment:
no, that is not how it works, @DmytroHryshyn I think it just creates a comment where to remove the props?

Reply:
@grzpab Yeah, you're right. Well in this case, we're doing something totally different than what's stated in the [vercel docs](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#static-site-generation-getstaticprops).

I'll update this to simply state that we're adding a comment indicating where to remove the props.